fireant/slicer/widgets/reacttable.py

```diff
@@ -83,8 +83,13 @@ def __init__(self, item, reference):
             self.key = reference_key(item, reference)
             self.label = reference_label(item, reference)
 
-        self.prefix = item.prefix
-        self.suffix = item.suffix
+        if reference is not None and reference.delta_percent:
```

Comment:
Maybe you can move this logic into fireant.slicer.references so i can be reused across other widgets

```
def reference_prefix(metric, reference):
    """
    Return the prefix for a metric displayed for a reference (or no Reference)

    :return:
        A string that is used as the prefix for a reference metric.
    """
    if reference is not None and reference.delta_percent:
        return None
    return metric.prefix
```

Reply:
same for suffix